<ui version="4.0" >
 <class>WPEditAccountBase</class>
 <widget class="QWidget" name="WPEditAccountBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Account Preferences - WinPopup</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget10" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>1</hsizetype>
       <vsizetype>1</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Basi&amp;c Setup</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Account Information</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label1" >
              <property name="toolTip" >
               <string>The hostname under which you want to send WinPopup messages.</string>
              </property>
              <property name="whatsThis" >
               <string>The host name under which you want to send WinPopup messages. Note that you do not have to specify the true hostname in order to send messages, but you do if you want to receive them.</string>
              </property>
              <property name="text" >
               <string>Hos&amp;tname:</string>
              </property>
              <property name="buddy" >
               <cstring>mHostName</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="mHostName" >
              <property name="toolTip" >
               <string>The hostname under which you want to send WinPopup messages.</string>
              </property>
              <property name="whatsThis" >
               <string>The host name under which you want to send WinPopup messages. Note that you do not have to specify the true hostname in order to send messages, but you do if you want to receive them.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="doInstallSamba" >
              <property name="toolTip" >
               <string>Install support into Samba to enable this service.</string>
              </property>
              <property name="whatsThis" >
               <string>Install support into Samba to enable this service.</string>
              </property>
              <property name="text" >
               <string>I&amp;nstall Into Samba</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>1</hsizetype>
           <vsizetype>1</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Information</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel12" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>To receive WinPopup messages sent from other machines, the hostname above must be set to this machine's hostname.</string>
            </property>
            <property name="textFormat" >
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel1_3" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>The samba server must be configured and running.</string>
            </property>
            <property name="textFormat" >
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>"Install into Samba" is a simple method to create the directory for the temporary message files and configure your samba server.&lt;br>
However, the recommended way is to ask your administrator to create this directory ('mkdir -p -m 0777 /var/lib/winpopup') and for you to add
'message command = _PATH_TO_/winpopup-send %s %m %t &amp;' (substitute _PATH_TO_ by the real path) to the [global] section in your smb.conf.</string>
            </property>
            <property name="textFormat" >
             <enum>Qt::RichText</enum>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage" >
      <attribute name="title" >
       <string>S&amp;ystem</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Protocol Preferences</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="1" >
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QSpinBox" name="mHostCheckFreq" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>5</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="textLabel6" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>3</hsizetype>
                  <vsizetype>5</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>second(s)</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="textLabel4" >
              <property name="text" >
               <string>Host check frequency:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="KUrlRequester" name="mSmbcPath" />
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="textLabel1" >
              <property name="text" >
               <string>Path to 'smbclient' executable:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Information</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel2_2" >
            <property name="text" >
             <string>&lt;i>These options apply to all WinPopup accounts.&lt;/i></string>
            </property>
            <property name="textFormat" >
             <enum>Qt::RichText</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>135</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelStatusMessage" >
     <property name="text" >
      <string/>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSpinBox</class>
   <extends></extends>
   <header>knuminput.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends></extends>
   <header>kurlrequester.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget10</tabstop>
  <tabstop>mHostName</tabstop>
  <tabstop>doInstallSamba</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
